scripts/ci/publish.js

```diff
@@ -9,15 +9,16 @@ const {
   TAG,
   BRANCH
 } = process.env;
-const releaseName = TAG || BRANCH;
-const PouchDB = require('pouchdb-core');
-PouchDB.plugin(require('pouchdb-adapter-http'));
+const versions = require('../build/versions');
 
+const releaseName = TAG || versions.escapeBranchName(BRANCH);
 if (!releaseName) {
   console.log('Not a tag or a branch so not publishing. Most likely this is a PR build which is merged with master');
   process.exit(0);
 }
 
+const PouchDB = require('pouchdb-core');
+PouchDB.plugin(require('pouchdb-adapter-http'));
```

Comment:
question: Why were the imports shifted downwards?

Reply:
The release name check exits the script in certain cases where it's not necessary to run, so I decided there was no point setting up Pouch if the script was about to exit.